### PR TITLE
Update index_cn.rst

### PR DESCRIPTION
**Pull request by AdamzNV**
This is Adam, NVIDIA Product Manager for PaddlePaddle. To ensure PP developers can get the installation manual easier, I update the description of NVIDIA NGC PaddlePaddle Container here. 

Many thanks.

### CODE REVIEW

**Comment by AdamzNV** [comment]
++ @onecatcn

File: docs/install/index_cn.rst [hunk 1]
```diff
@@ -193,7 +194,14 @@
     `Windows 下的 PIP 安装 <pip/windows-pip.html>`_
 
 
-**第二种安装方式：使用源代码编译安装**
+**第二种安装方式：使用容器安装**
+
+- 推荐使用 `NVIDIA 飞桨容器安装 <https://www.paddlepaddle.org.cn/documentation/docs/zh/install/install_NGC_PaddlePaddle_ch.html>`_
+     1. 适配最新版的CUDA
```
Comment (sunzhongkai588) [inline docs/install/index_cn.rst]: `CUDA`前加空格，注意中英文间请加空格～，下面的`Ubuntu OS(18.04)`和`NVIDIA `也一样

**Review by sunzhongkai588 — COMMENTED** [review]
增加空格～

File: docs/install/index_cn.rst (hunk 1 above)
Comment (AdamzNV) [reply docs/install/index_cn.rst]: Finalized

**Pull request closed without merging**